plugins/mcp/plugin_test.go

```
@@ -0,0 +1,691 @@
+				fmt.Printf("  Tool call %d: %s\n", i+1, *toolCall.Function.Name)
+			}
+		}
+	})
+
+	// Cleanup
+	client.Cleanup()
+	fmt.Println("\n=== TEST COMPLETED SUCCESSFULLY ===")
+}
+
+// Helper function to print choices as JSON
+func printChoicesAsJSON(t *testing.T, choices []schemas.BifrostResponseChoice) {
```

golangci-lint (1.64.8)

func `printChoicesAsJSON` is unused

(unused)